src/__tests__/chains.test.ts

```diff
@@ -0,0 +1,20 @@
+import express, { Express } from 'express';
+import request from 'supertest';
+
+import router from '../routes';
+
+describe('GET /chains', () => {
+  let app: Express;
+
+  beforeAll(() => {
+    app = express();
+    app.use(express.json());
+    app.use('/api', router);
```

Comment:
we don't have `/api` prefix on our endpoints, so i think it's better to leave it out here too.

```suggestion
    app.use('/, router);
```